physics/coulombs_law.py

```diff
@@ -0,0 +1,37 @@
+"""
+The law states that the magnitude of the electrostatic force of attraction or repulsion between two point charges
+is directly proportional to the product of the magnitudes of charges and inversely proportional to the square of the distance between them.
+
+Coulomb studied the repulsive force between bodies having electrical charges of the same sign.
+
+F  = k*q1*q2/ r^2
+
+k is proportionality constant and equals 1/4πε0.
```

Comment:
```suggestion
k is the Coulomb constant and equals 1/(4π * ε0).
```
According to [Wikipedia](https://en.wikipedia.org/wiki/Coulomb_constant), $k$ is called the Coulomb constant

Added parentheses for clarity, because otherwise it looks like $\frac{1}{4} \pi \varepsilon_0$ and not $\frac{1}{4 \pi \varepsilon_0}$